lib/discordrb/bot.rb

```diff
@@ -883,6 +883,7 @@ def update_guild_member(data)
     def delete_guild_member(data)
       server_id = data['guild_id'].to_i
       server = self.server(server_id)
+      return unless server # Server was deleted on GUILD_DELETE. See #619
```

Comment:
This code is pretty self-explanatory to most readers of the internals - the comment isn't necessary. It's well known that `bot.server(id)` can return `nil` on a cache miss, which could be for any reason, including from processing a `GUILD_DELETE`.
```suggestion
      return unless server
```